.gitignore

```diff
@@ -6,6 +6,7 @@
 
 # Installation artifacts
 src
+requirements/*.txt
```

Comment:
BTW, this is a standard part of the Django app cookicutter: https://github.com/edx/cookiecutter-django-app